distros/librealsense2-dont-build-nlohmann-json.patch

```diff
@@ -0,0 +1,17 @@
+*** a/third-party/CMakeLists.txt	2024-07-05 11:40:28.704390535 +0900
+--- b/third-party/CMakeLists.txt	2024-07-05 11:44:37.331048514 +0900
+***************
+*** 1,6 ****
+  string(REPLACE ${PROJECT_SOURCE_DIR}/ "" _rel_path ${CMAKE_CURRENT_LIST_DIR})
+  
+! include(CMake/external_json.cmake)
+  
+  add_subdirectory( "${CMAKE_CURRENT_LIST_DIR}/rsutils" )
+  
+--- 1,6 ----
+  string(REPLACE ${PROJECT_SOURCE_DIR}/ "" _rel_path ${CMAKE_CURRENT_LIST_DIR})
+  
+! # include(CMake/external_json.cmake)
```

Comment:
We prefer not to have patches here, but instead fetch them from upstream PRs or development commits. This is, of course, not an upstreamable patch. If we want it here, I'd prefer so called unified diff format, which is more readable. It can be produced with `git diff --no-index` or `diff -u`. However, it this case, we can do without the patch. We can use [`substituteInPlace`](https://nixos.org/manual/nixpkgs/unstable/#fun-substituteInPlace) in the override:

```
...
postPatch = postPatch + ''
  substituteInPlace third-party/CMakeLists.txt --replace-fail 'include(CMake/external_json.cmake)' ""
'';
```

Reply:
sg; I've replaced the patch with the postPatch.